spark/v3.5/spark/src/test/java/org/apache/iceberg/spark/actions/TestRewritePositionDeleteFilesAction.java

```diff
@@ -283,6 +286,12 @@ public void testRewriteFilter() throws Exception {
     List<Object[]> actualDeletes = deleteRecords(table);
     assertEquals("Rows must match", expectedRecords, actualRecords);
     assertEquals("Position deletes must match", expectedDeletes, actualDeletes);
+
+    sql("set %s = true", SQLConf.CASE_SENSITIVE().key());
+    assertThatThrownBy(
```

Comment:
can we use `withSQLConf` as this might cause if we use re:use spark session ?
-https://github.com/apache/iceberg/blob/36140b819c60743b39176e28a63ab588df445329/spark/v3.5/spark/src/test/java/org/apache/iceberg/spark/TestBase.java#L184

Reply:
Changed. Thanks!